### PR TITLE
Payment Flow: Amounts, contribution types

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## Why are you doing this?

In this PR, I'm adding the rendering logic, given a `countryGroupId` and a `contributionType`, of the one-off/monthly choice and their respective amounts per country group. The only interesting change is thus in the JSX for contribution type and amount.

Amounts, currencies and contribution types: These information may exist somewhere, please let me know if I can easily embed them in this code.

In a follow-up PR, I will:

- add these information to the store and update from there (right now it is hard-coded)
- update the state based on user interactions

[**Trello Card**](https://trello.com/c/ALFY3EbV/775-npf-display-amounts-and-currencies-per-country-group)

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
> please let me know if I can easily embed them in this code.

If they are currently part of a test (like the annual contribution type), I'd prefer not to use them for now.

**Comment by joelochlann** [comment]
`assets/helpers/contributions.js`

**Comment by joelochlann** [comment]
This could also be useful: `assets/helpers/internationalisation/countryGroup.js`

**Comment by joelochlann** [comment]
Personally I think it makes sense to use the values in https://github.com/guardian/support-frontend/blob/master/assets/helpers/contributions.js now rather than deferring that bit of work - otherwise this PR contains a lot of duplication of existing code which could easily trip someone up.

Not to mention the fact that if the default amounts change on the current page but are not kept in sync here, it would affect the outcome of this test

**Comment by regiskuckaertz** [comment]
@joelochlann fair enough, will update using that module.

File: assets/pages/contributions-landing/newContributionsLanding.jsx [hunk 1]
```diff
@@ -87,35 +177,48 @@ const content = (
...
-            <li className="form__radio-group__item">
-              <input id="contributionAmount-100" className="form__radio-group__input" type="radio" name="contributionAmount" value="100" />
-              <label htmlFor="contributionAmount-100" className="form__radio-group__label" aria-label="100 US Dollars">$100</label>
-            </li>
-            <li className="form__radio-group__item">
-              <input id="contributionAmount-250" className="form__radio-group__input" type="radio" name="contributionAmount" value="250" />
-              <label htmlFor="contributionAmount-250" className="form__radio-group__label" aria-label="250 US Dollars">$250</label>
-            </li>
+            {selectedCountryGroup.amounts[contributionType].map((a, i) => (
+              <li className="form__radio-group__item">
```
Comment (jranks123) [inline assets/pages/contributions-landing/newContributionsLanding.jsx]: It would be good to name `a` and `i` something more descriptive

File: assets/pages/contributions-landing/newContributionsLanding.jsx [hunk 2]
```diff
@@ -87,35 +177,48 @@ const content = (
...
+                <input
+                  id={`contributionAmount-${a}`}
+                  className="form__radio-group__input"
+                  type="radio"
+                  name="contributionAmount"
+                  value={a}
+                  checked={i === 0}
+                />
+                <label htmlFor={`contributionAmount-${a}`} className="form__radio-group__label" aria-label={formatAmount(a, true)}>
```
Comment (jranks123) [inline assets/pages/contributions-landing/newContributionsLanding.jsx]: It would be good to name `a` something more descriptive

File: assets/pages/contributions-landing/newContributionsLanding.jsx (hunk 1 above)
Comment (regiskuckaertz) [reply assets/pages/contributions-landing/newContributionsLanding.jsx]: I think `i` is idiomatic enough but you're right about `a`.

**Comment by regiskuckaertz** [comment]
@jranks123 thanks for your review 🙏

**Comment by regiskuckaertz** [comment]
@joelochlann have used the helper module. There's loads of other useful functions in there, will integrate more as I go along.

File: assets/pages/contributions-landing/newContributionsLanding.jsx [hunk 3]
```diff
@@ -43,43 +33,147 @@ const defaultContributeCopy = `
...
 `;
 
+type CountryMetaData = {
+  headerCopy: string,
+  contributeCopy: string,
+  currency: Object,
+  contribution: Object
+};
+
 const countryGroupSpecificDetails: {
```
Comment (joelochlann) [inline assets/pages/contributions-landing/newContributionsLanding.jsx]: You should be able to get rid of this entire object using stuff in `assets/helpers/` - happy for that to be in a separate PR but I definitely think it should be done before putting the test live

Comment (regiskuckaertz) [reply assets/pages/contributions-landing/newContributionsLanding.jsx]: Agreed.

**Comment by regiskuckaertz** [comment]
@Ap0c thanks, I'll write this down and raise a follow-up
